src/components/view/ContactIntake/index.js

```diff
@@ -97,20 +93,18 @@ class IntakeForm extends Component {
     // or locally within state. so I'm abstracting this call to make it clear what data
     // we send in event creation
     packageFormDataForSubmission() {
-        const outreach = this.state.showOutreach ? {
+        const visitOrOutreach = {
+            outreach: this.state.outreach,
+            referral: this.state.referral,
             syringesGiven: this.state.syringesGiven,
             syringesTaken: this.state.syringesTaken,
             narcanWasOffered: this.state.narcanWasOffered,
             narcanWasTaken: this.state.narcanWasTaken,
             enrollment: this.state.enrollment,
             numberOfOthersHelping: this.state.numberOfOthersHelping,
-        } : null;
-
-         const standard = {
-             referral: this.state.referral,
-             location: this.state.eventLocation,
-             contactUid: this.state.uid,
-         }
+            location: this.state.eventLocation,
```

Comment:
I think I may have missed removing this. but we can get rid of it since it's going in via #39